src/Tabs/Tabs.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Nav } from '../index'
+
+/**
+ * Tabs Component for Patternfly React
+ */
+const Tabs = props =>
+  <Nav
+    onSelect={props.onSelect}
+    className={props.className}
+    activeKey={props.activeKey}
+    {...props}
+  >
+    {props.children}
+  </Nav>
+
+Tabs.propTypes = {
+  /** tab selected callback */
+  onSelect: PropTypes.func,
+  /** nav tabs class */
+  className: PropTypes.string,
+  /** sets the active tab key */
+  activeKey: PropTypes.number,
+  /** children nodes  */
+  children: PropTypes.node
+}
+Tabs.defaultProps = {
+  className: 'nav nav-tabs'
+}
```

Comment:
@jgiardino i've updated the logic here in `Tabs` to check for a `TabRowContents` rendered child and apply the `pf-tabrow` class if found. Typically i've used state/props to pass down attributes, but since we are modularizing this for the consumer, I think it's OK (I think React Bootstrap does this sort of thing after reviewing it further).

I'm not sure if this is an acceptable React pattern though. @jtomasek any thoughts on this approach?